### ISSUE TITLE
want to know how to interpolate

### ISSUE THREAD

**Issue by izshuang** [wywu/LAB#39]
Hi  
   i want to know how to get the boundary line, i use interp1d() to get the line ,but i get some false in some pic.

**Comment by meiluzhu** [comment]
Yes, I also met this problem @izshuang using any interpolation algorithms
